lib/svgeez.rb

```diff
@@ -2,12 +2,20 @@
 require 'listen'
 require 'logger'
 require 'mkmf'
+require 'securerandom'
 
 require 'svgeez/version'
+
 require 'svgeez/command'
 require 'svgeez/commands/build'
 require 'svgeez/commands/watch'
-require 'svgeez/sprite_builder'
+
+require 'svgeez/builder'
+require 'svgeez/destination'
+require 'svgeez/optimizer'
+require 'svgeez/source'
+require 'svgeez/svg_element'
+require 'svgeez/symbol_element'
```

Comment:
Created a handful of new, relatively self-contained classes for major pieces of functionality.